api/envoy/config/core/v3/substitution_format_string.proto

```diff
@@ -57,5 +60,29 @@ message SubstitutionFormatString {
     //  }
     //
     google.protobuf.Struct json_format = 2 [(validate.rules).message = {required: true}];
+
+    // A DataSource variant for text_format.
+    //
+    // The same semantics apply to the string/ that the referenced DataSource contains.
```

Comment:
> string/

Is that `/` some form of markup that I don't understand?

Reply:
that is a typo